src/KeyedDistributions.jl

```diff
@@ -91,6 +91,9 @@ const MvNormalLike = Union{MvNormal, KeyedMvNormal}
 const KeyedGenericMvTDist = KeyedDistribution{Multivariate, Continuous, <:GenericMvTDist}
 const MvTLike = Union{GenericMvTDist, KeyedGenericMvTDist}
 
+const KeyedMixtureModel = KeyedDistribution{<:VariateForm, <:ValueSupport, <:AbstractMixtureModel}
+const KeyedMixtureLike = Union{MixtureModel, KeyedMixtureModel}
```

Comment:
Naming pattern elsewhere seems to follow"
```suggestion
const MixtureLike = Union{MixtureModel, KeyedMixtureModel}
```